docs/administrator/security/certificate-configuration.md

```diff
@@ -0,0 +1,334 @@
+  ```
+
+- **karmada-scheduler** mounts the secrets `karmada-cert-secret` and `kubeconfig` and uses the certificates for:
+
+  ```yaml
+              - --kubeconfig=/etc/kubeconfig
+              - --scheduler-estimator-ca-file=/etc/karmada/pki/ca.crt
+              - --scheduler-estimator-cert-file=/etc/karmada/pki/karmada.crt
+              - --scheduler-estimator-key-file=/etc/karmada/pki/karmada.key
+  ```
+
+- **karmada-descheduler** mounts the secrets `karmada-cert-secret` and `kubeconfig` and uses the certificates for:
+
+  ```yaml
+              - --kubeconfig=/etc/kubeconfig
+              - --scheduler-estimator-ca-file=/etc/karmada/pki/ca.crt
+              - --scheduler-estimator-cert-file=/etc/karmada/pki/karmada.crt
+              - --scheduler-estimator-key-file=/etc/karmada/pki/karmada.key
+  ```
+
+- **karmada-metrics-adapter** mounts the secrets `karmada-cert-secret` and `kubeconfig` and uses the certificates for:
+
+  ```yaml
+              - --kubeconfig=/etc/kubeconfig
+              - --authentication-kubeconfig=/etc/kubeconfig
+              - --authorization-kubeconfig=/etc/kubeconfig
+              - --client-ca-file=/etc/karmada/pki/ca.crt
+              - --tls-cert-file=/etc/karmada/pki/karmada.crt
+              - --tls-private-key-file=/etc/karmada/pki/karmada.key
+  ```
+
+- **karmada-search** mounts the secrets `karmada-cert-secret` and `kubeconfig` and uses the certificates for:
+
+  ```yaml
+              - --kubeconfig=/etc/kubeconfig
+              - --authentication-kubeconfig=/etc/kubeconfig
+              - --authorization-kubeconfig=/etc/kubeconfig
+              - --etcd-cafile=/etc/karmada/pki/etcd-ca.crt
+              - --etcd-certfile=/etc/karmada/pki/etcd-client.crt
+              - --etcd-keyfile=/etc/karmada/pki/etcd-client.key
+              - --tls-cert-file=/etc/karmada/pki/karmada.crt
+              - --tls-private-key-file=/etc/karmada/pki/karmada.key
+  ```
+
+- **karmada-webhook** mounts the secrets `webhook-cert` and `kubeconfig` and uses the certificates for:
+
+  ```yaml
+              - --kubeconfig=/etc/kubeconfig
+              - --cert-dir=/var/serving-cert
+  ```
+
+#### Checking Certificate Expiry Dates
+
+You can use the command `openssl x509 -noout -dates -in path/to/cert` to extract the before and after dates of the certificate.
+
+```bash
+$ openssl x509 -noout -dates -in path/to/cert
+notBefore=Aug 24 16:00:00 2024 GMT
+notAfter=Aug 29 16:00:00 2024 GMT
+```
+
+The output indicates that this certificate becomes valid at 16:00 on August 24, 2024, and expires at 16:00 on August 29, 2024.
+
+
+### **Certificate Replacement**
+
+When a business certificate expires or is about to expire, you can manually replace it with a new certificate to extend its validity.
+
+#### generate a new certificate
+
+1. **Determine the issuing CA of the expired certificate.** Refer to the **Karmada Certificate Overview** for the relationship between business certificates and their issuing CAs.

```

Comment:
```suggestion
1. **Determine the issuing CA of the expired certificate.** Refer to the [Karmada Certificate Overview](#karmada-certificate-overview) for the relationship between business certificates and their issuing CAs.
```
It's better to provide a link rather than a bold text.